docs/reference/modules/cluster/remote-clusters-api-key.asciidoc

```diff
@@ -0,0 +1,177 @@
+[[remote-clusters-api-key]]
+=== Add remote clusters using API key authentication
+
+coming::[8.10]
+beta::[]
+
+API key authentication enables a local cluster to authenticate itself with a
+remote cluster via a cross-cluster API key. The API key needs to be created by
```

Comment:
We can link to the [CreateCrossClusterApiKey](https://www.elastic.co/guide/en/elasticsearch/reference/master/security-api-create-cross-cluster-api-key.html) doc page.